packages/core/src/saml-applications/routes/index.ts

```diff
@@ -266,9 +276,9 @@ export default function samlApplicationRoutes<T extends ManagementApiRouter>(
     koaGuard({
       params: z.object({ id: z.string() }),
       status: [200, 400, 404],
-      response: z.object({
-        certificate: z.string(),
-        fingerprints: certificateFingerprintsGuard,
+      response: samlApplicationSecretResponseGuard.pick({
```

Comment:
What is the use case of this API, do we need to return the fingerprints here? Also should be `fingerprint`?

Reply:
Discussed offline, updated